### PR TITLE
migration: Add a new param

### PR DESCRIPTION
**Pull request by cliping**
Add a param to check guest state on source host.

Signed-off-by: cliping <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result: https://github.com/autotest/tp-libvirt/pull/4252

**Comment by cliping** [comment]
@dzhengfy  @Yingshun , please take a look, thanks.

File: virttest/migration.py [hunk 1]
```diff
@@ -59,40 +59,47 @@ def post_migration_check(self, vms, params, uptime=None, uri=None):
...
         :param vms: VM objects of migrating vms
         :param uptime: uptime dict of vms before migration
-        :param uri: target virsh uri
+        :param dest_uri: target virsh uri
+        :param src_uri: source virsh uri
         :return: updated dict of uptime
         """
-        vm_state = params.get("virsh_migrated_state", "running")
+        vm_state_dest = params.get("virsh_migrate_dest_state", "running")
+        vm_state_src = params.get("virsh_migrate_src_state")
```
Comment (dzhengfy) [inline virttest/migration.py:67]: Could we have 'Shut off' as default value for `vm_state_src`?

**Review by dzhengfy — COMMENTED** [review]
Other LGTM

File: virttest/migration.py (hunk 1 above)
Comment (cliping) [reply virttest/migration.py]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

File: virttest/migration.py [hunk 2]
```diff
@@ -59,40 +59,47 @@ def post_migration_check(self, vms, params, uptime=None, uri=None):
...
         :param vms: VM objects of migrating vms
         :param uptime: uptime dict of vms before migration
-        :param uri: target virsh uri
+        :param dest_uri: target virsh uri
+        :param src_uri: source virsh uri
         :return: updated dict of uptime
         """
-        vm_state = params.get("virsh_migrated_state", "running")
+        vm_state_dest = params.get("virsh_migrate_dest_state", "running")
+        vm_state_src = params.get("virsh_migrate_src_state", "shut off")
```
Comment (dzhengfy) [inline virttest/migration.py:67]: Just a reminder. 
In existing tp-libvirt cases, some python files already use 'virsh_migrate_src_state' for another purpose like the prepared vm state before migration. So we need to be careful if those files use this function in future and we need rename the variable if any.

Comment (dzhengfy) [inline virttest/migration.py:67]: Just a reminder.
In existing tp-libvirt cases, some python files already use 'virsh_migrate_src_state' for another purpose like the prepared vm state before migration. So we need to be careful if those files use this function in future and we need rename the variable if any. But so far so good.

**Review by dzhengfy — APPROVED** [review]

**Pull request merged**